pypesto/profile/profile_next_guess.py

```diff
@@ -263,37 +262,19 @@ def par_extrapol(step_length):
     next_obj = problem.objective(problem.get_reduced_vector(next_x))
 
     # iterate until good step size is found
-    if next_obj_target < next_obj:
-        # The step is rather too long
-        return do_line_search(
-            next_x,
-            step_size_guess,
-            'decrease',
-            par_extrapol,
-            next_obj,
-            next_obj_target,
-            clip_to_minmax,
-            clip_to_bounds,
-            par_index,
-            problem,
-            options,
-        )
-
-    else:
-        # The step is rather too short
-        return do_line_search(
-            next_x,
-            step_size_guess,
-            'increase',
-            par_extrapol,
-            next_obj,
-            next_obj_target,
-            clip_to_minmax,
-            clip_to_bounds,
-            par_index,
-            problem,
-            options,
-        )
```

Comment:
specific reason to remove this? Wouldn't this now prevent the stepsize to get bigger if the landscape allows for this?

Reply:
Handled by `"decrease" if next_obj_target < next_obj else "increase",`.
Same functionality as before, just getting rid of 12 redundant lines.

Reply:
yes, only saw that now, sorry.